lib/widgets/drawer.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+
+class SidebarDrawer extends StatefulWidget {
+  @override
+  _SidebarDrawerState createState() => _SidebarDrawerState();
+}
+
+class _SidebarDrawerState extends State<SidebarDrawer> {
+  Widget build(BuildContext context) {
+    //DisabilityNotifier disabilityNotifier = Provider.of<DisabilityNotifier>(context);
```

Comment:
Why is this commented out? Should it be there?

Reply:
Deleted this comment.